pkg/kubelet/kubelet.go

```diff
@@ -203,7 +203,6 @@ type Builder func(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	registerNode bool,
 	registerWithTaints []api.Taint,
 	allowedUnsafeSysctls []string,
-	containerized bool,
```

Comment:
You should also remove it from the Kubelet struct in this file.

Reply:
OK, I have removed it.